website/homepage/src/index/demos_template.html

```diff
@@ -15,6 +15,16 @@
              onclick="load_demo(3)">
             Image Segmentation
         </div>
+        <div class="demo-tab hover:text-gray-800 cursor-pointer px-3 py-1"
+             demo="4"
+             onclick="load_demo(4)">
+            Time Series Forecasting With Prophet
```

Comment:
Thoughts on shortening the titles so that they fit better and roughly match the length of the titles of the first 3 demos?
```suggestion
            Time Series Forecasting
```

Reply:
Sure thing! I was just trying to use eye catching terms lol